src/buildercore/trop.py

```diff
@@ -305,27 +305,34 @@ def render_rds(context, template):
     ]
     lmap(template.add_output, outputs)
 
-def render_ext_volume(context, context_ext, template, node=1):
+def render_ext_volume(context, context_ext, template, actual_ec2_instances, node=1):
     vtype = context_ext.get('type', 'standard')
+    # TODO: remove if not in use
     if vtype == 'ssd':
-        LOG.warn("deprecated value of 'ssd' used in aws.ext.type: %s", context['stackname'])
-        vtype = 'gp2'
+        raise RuntimeError("deprecated value of 'ssd' used in aws.ext.type: %s", context['stackname'])
```

Comment:
if you're now failing on it, then it's obsoleted rather than deprecated.

a better message might be "Obsolete value of 'ssd' used in 'aws.ext.type' on stack '%s'. This should be a value of 'gp2'."

Reply:
I found out this is not in use anymore in any file, so time to remove the check

Reply:
good call